package.nix

```diff
@@ -37,27 +38,29 @@ let
       ];
 
       preCheck = ''
-        ${lib.getExe pkgs.cargo} clippy
+        ${lib.getExe cargo} clippy
 
         # Stop the Nix command from trying to create /nix/var/nix/profiles.
         #
         # https://nix.dev/manual/nix/2.24/command-ref/new-cli/nix3-profile#profiles
         export NIX_STATE_DIR=$TMPDIR
       '';
-    }
-  ) { };
+    };
 in
-{
-  inherit system-manager-unwrapped;
-  system-manager =
-    pkgs.runCommand "system-manager"
+    runCommand "system-manager"
       {
-        nativeBuildInputs = [ pkgs.pkgsBuildHost.makeBinaryWrapper ];
```

Comment:
Just want to make sure this was intentional, I'm not sure if we have any tests for cross compilation and a contributor just added this change.

Reply:
AFAIU, since `makeBinaryWrapper` is supplied through `callPackage` now, splicing should take care of picking the correct package. But let me check and get back to you on that.

Reply:
Yep, this branch (specifically d84a47bbf61299ce25a4b5b632cda0f0da92ace2) seems to behave as expected for cross-compilation.

Reply:
Thanks @lierdakil 🙏🏻

Reply:
sweet, thanks for checking @lierdakil